### PR TITLE
Add documentation for mock interview - interviewee

### PR DESCRIPTION
**Pull request by abiisnn**
# What does this PR do?
It adds documentation for scheduling a mock interview (for the interviewee) in **Spanish**.  There are 3 new sections:

- Log in
- Upload profile
- Schedule mock interviews

# Evidence

Evidencia

### CODE REVIEW

**Review by searleser97 — APPROVED** [review]
Fantastique !

File: i18n/es/docusaurus-plugin-content-docs/current/yaos/interviewee/mock-interview.mdx [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+A continuación, selecciona la opción **Continue with Google** y elige la cuenta de google con la que quieres entrar.
+
+![Using google](/img/docs/yaos/interviewee/mock-interview/using-google.svg)
+
+¡Listo! 🎉 Ahora podrás ver la página para actulizar tu NutriPerfil.
+
+![Update profile page](/img/docs/yaos/interviewee/mock-interview/update-profile-page.png)
+
+## :clipboard:✏️Actualizar perfil
+Para empezar, deberás actulizar tu **NutriProfile** (aka tu información básica). Elige tu escuela y sube tu resume **en formato pdf**.
```
Comment (searleser97) [inline i18n/es/docusaurus-plugin-content-docs/current/yaos/interviewee/mock-interview.mdx:20]: actualizar

File: i18n/es/docusaurus-plugin-content-docs/current/yaos/interviewee/mock-interview.mdx [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+### Información acerca del tipo de entrevista
+
+![Schedule mock](/img/docs/yaos/interviewee/mock-interview/schedule-mock.svg)
+
+
+1. **Position:** ¿A qué posición estás aplicando? (Explore/STEP/Internship/FUll Time).
+2. **Role applying to:** ¿Con qué rol quieres ser entrevistado? (Software Engineer/ Machine Learning, etc).
+3. **Number of Interviews:** ¿Cuántas entrevistas te gustaría hacer en el rango de tiempo que estás registrando? (Tienes derecho de hasta 3, considera que es basado en disponibilidad de entrevistadores, por eso te pedimos que tu rango de tiempo sea **flexible**).
+4. **Programming Languages to use in the interview:** Selecciona con el que te sientas más cómodo.
+5. **Company applying to:** Elige la compañia con para la que estás aplicando.
```
Comment (searleser97) [inline i18n/es/docusaurus-plugin-content-docs/current/yaos/interviewee/mock-interview.mdx:53]: "con para" ? no seria solo "para la que..."

File: i18n/es/docusaurus-plugin-content-docs/current/yaos/interviewee/mock-interview.mdx [hunk 3]
```diff
@@ -0,0 +1,87 @@
+---
+id: mock-interview
+title: ¿Cómo agendar una entrevista?
+---
+
+## :bust_in_silhouette: Iniciar sesión
+Para iniciar sesión, necesitas una correo de GMAIL (usuario@gmail.com), ya que Proyecto Nutria solo soporta inicio de sesión con Google. Para Iniciar sesión, da click en el boton **LOGIN**.
```
Comment (akotadi) [inline i18n/es/docusaurus-plugin-content-docs/current/yaos/interviewee/mock-interview.mdx:7]: nit: botón

File: i18n/es/docusaurus-plugin-content-docs/current/yaos/interviewee/mock-interview.mdx [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+
+![Login](/img/docs/yaos/interviewee/mock-interview/login.svg)
+
+A continuación, selecciona la opción **Continue with Google** y elige la cuenta de google con la que quieres entrar.
+
+![Using google](/img/docs/yaos/interviewee/mock-interview/using-google.svg)
+
+¡Listo! 🎉 Ahora podrás ver la página para actulizar tu NutriPerfil.
```
Comment (akotadi) [inline i18n/es/docusaurus-plugin-content-docs/current/yaos/interviewee/mock-interview.mdx:15]: nit: actualizar

File: i18n/es/docusaurus-plugin-content-docs/current/yaos/interviewee/mock-interview.mdx [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+Para empezar, deberás actulizar tu **NutriProfile** (aka tu información básica). Elige tu escuela y sube tu resume **en formato pdf**. 
+
+:warning: **Importante:** Si no llenas ambos campos, no podrás seguir adelante.
+
+Después da click en **UPLOAD PROFILE**.
+
+![Update profile page](/img/docs/yaos/interviewee/mock-interview/update-profile-data.jpg)
+
+
+¡Listo! ahora puedes *Agendar Mock interviews*, *Ver tus proximas entrevistas* y *Ver el historial de tus entrevistas*.
```
Comment (akotadi) [inline i18n/es/docusaurus-plugin-content-docs/current/yaos/interviewee/mock-interview.mdx:29]: nit: próximas

File: i18n/es/docusaurus-plugin-content-docs/current/yaos/interviewee/mock-interview.mdx [hunk 6]
```diff
@@ -0,0 +1,87 @@
...
+Podrás ver la pantalla de **Schedule Interview**. 
+
+### Información acerca del tipo de entrevista
+
+![Schedule mock](/img/docs/yaos/interviewee/mock-interview/schedule-mock.svg)
+
+
+1. **Position:** ¿A qué posición estás aplicando? (Explore/STEP/Internship/FUll Time).
+2. **Role applying to:** ¿Con qué rol quieres ser entrevistado? (Software Engineer/ Machine Learning, etc).
+3. **Number of Interviews:** ¿Cuántas entrevistas te gustaría hacer en el rango de tiempo que estás registrando? (Tienes derecho de hasta 3, considera que es basado en disponibilidad de entrevistadores, por eso te pedimos que tu rango de tiempo sea **flexible**).
```
Comment (akotadi) [inline i18n/es/docusaurus-plugin-content-docs/current/yaos/interviewee/mock-interview.mdx:51]: basado en *la* disponibilidad

File: i18n/es/docusaurus-plugin-content-docs/current/yaos/interviewee/mock-interview.mdx [hunk 7]
```diff
@@ -0,0 +1,87 @@
...
+4. **Programming Languages to use in the interview:** Selecciona con el que te sientas más cómodo.
+5. **Company applying to:** Elige la compañia con para la que estás aplicando.
+
+
+
+### :clock1: Agregar TUS horarios disponibles para prácticar
+Para agregar tus horarios disponibles para prácticar Mock interviews, da click en el botón **Add availability range**. ***Puedes agregar más de un rango de horario.*
```
Comment (akotadi) [inline i18n/es/docusaurus-plugin-content-docs/current/yaos/interviewee/mock-interview.mdx:58]: nit: practicar

File: i18n/es/docusaurus-plugin-content-docs/current/yaos/interviewee/mock-interview.mdx [hunk 8]
```diff
@@ -0,0 +1,87 @@
...
+
+:warning: **Importante:** Tus horarios deben estar en **PT time**, tal como menciona la nota.
+
+![Hour range](/img/docs/yaos/interviewee/mock-interview/hour.svg)
+
+
+Considera que mientras más horarios des, hay más probabilidad de que un entrevistador te agende entrevista. 
+
+Cuando tu información acerca del tipo de entrevista y tus horarios esten completos, da click en el botón **SCHEDULE**. ¡Listo! Ahora solo queda esperar a que un entrevistador te agende una mock interview.
```
Comment (akotadi) [inline i18n/es/docusaurus-plugin-content-docs/current/yaos/interviewee/mock-interview.mdx:67]: nit: estén

File: i18n/es/docusaurus-plugin-content-docs/current/yaos/interviewee/mock-interview.mdx [hunk 9]
```diff
@@ -0,0 +1,87 @@
...
+
+## 💻 ¡Tienes una entrevista próxima!
+
+Cuando un entrevistador te agende una mock entrevista, recibirás un correo como el siguiente:
```
Comment (akotadi) [inline i18n/es/docusaurus-plugin-content-docs/current/yaos/interviewee/mock-interview.mdx:71]: *mock interview* o *entrevista de práctica*, creo que `mock entrevista` ya suena a mucho spanglish. xD

File: i18n/es/docusaurus-plugin-content-docs/current/yaos/interviewee/mock-interview.mdx [hunk 10]
```diff
@@ -0,0 +1,87 @@
...
+
+![Email interview](/img/docs/yaos/interviewee/mock-interview/email-interview.png)
+
+Para conocer los detalles de tu entrevista, entra al sistema y desde el menu principal selecciona **Incoming mock interviews**
```
Comment (akotadi) [inline i18n/es/docusaurus-plugin-content-docs/current/yaos/interviewee/mock-interview.mdx:75]: nit: menú

File: i18n/es/docusaurus-plugin-content-docs/current/yaos/interviewee/mock-interview.mdx [hunk 11]
```diff
@@ -0,0 +1,87 @@
...
+
+![Incoming interview dashboard](/img/docs/yaos/interviewee/mock-interview/dashboard-incoming.svg)
+
+Podrás ver el día, la hora, el link al documento de google para programar y el número de Room/Canal de voz al que debes ingresar en el discord de Proyecto Nutria.
+
+![Incoming interview](/img/docs/yaos/interviewee/mock-interview/incoming-mock.png)
+
+
+## ⏰💻 Día de la entrevista
+Entra al Discord de Proyecto Nutria, en el día y hora indicados, en el Room que se te indico. (Puedes encontrar los Rooms en la parte izquiera del discord).
```
Comment (akotadi) [inline i18n/es/docusaurus-plugin-content-docs/current/yaos/interviewee/mock-interview.mdx:85]: nit: indicó
nit: izquierda

**Review by akotadi — COMMENTED** [review]
A few comments, thanks for the guide. :)

**Review by searleser97 — APPROVED** [review]
LGTM

**Review by akotadi — CHANGES_REQUESTED** [review]
A few details for cleaning purposes, follow this guide to check the translations for buttons and navbar of Docosaurus:
https://github.com/Proyecto-Nutria/otter-guides#json-translations

Also, I tried to run the project in Linux and throw me this kind of errors:
image
And is because the images are with caps at the end (i.e. `.PNG`), so we need to change that.

Finally, erase the files that I included for example purposes:
https://github.com/Proyecto-Nutria/otter-guides/blob/abi/mock-interview-docs/docs/yaos/interviewee/about.mdx
https://github.com/Proyecto-Nutria/otter-guides/blob/abi/mock-interview-docs/i18n/es/docusaurus-plugin-content-docs/current/yaos/interviewee/about.mdx

As well as their reference:
https://github.com/Proyecto-Nutria/otter-guides/blob/abi/mock-interview-docs/sidebars.js#L26

After erasing you should also run the translations commands to see that everything is fine, probably will say that some things need to be erased, lemme know if you have any other problem.

File: i18n/es/docusaurus-plugin-content-docs/current/yaos/interviewee/login.mdx [hunk 12]
```diff
@@ -0,0 +1,16 @@
+---
+id: login
+title: Iniciar sesion
```
[the remark this answers is not in the document]
Comment (abiisnn) [reply i18n/es/docusaurus-plugin-content-docs/current/yaos/interviewee/login.mdx:3]: Done, let me know if I'm missing something.

**Review by akotadi — APPROVED** [review]
Awesome! Thanks for it.

**Pull request merged**